bin/compare_clusters.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "dotenv"
+Dotenv.load(".env")
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), "..", "lib"))
+require "bundler/setup"
+require "utils/ppnum"
+require "zinzout"
+require "cluster_overlap"
+require "cluster"
+
+require "rspec"
+require "rspec/expectations"
+require "rspec/matchers"
+
+include RSpec::Matchers
+
+Services.mongo!
+
+def overlap_line(overlap_hash)
+  [overlap_hash[:cluster_id],
+   overlap_hash[:volume_id],
+   overlap_hash[:member_id],
+   overlap_hash[:copy_count],
+   overlap_hash[:brt_count],
+   overlap_hash[:wd_count],
+   overlap_hash[:lm_count],
+   overlap_hash[:access_count]].join("\t")
+end
+
+if __FILE__ == $PROGRAM_NAME
+  logger = Services.logger
+
+  ARGF.each_line do |line|
+    item_id = line.strip
+
+    #
+    cluster = Cluster.where("ht_items.item_id": item_id).first
+
+    new_htitems = cluster.ht_items.map(&:item_id).sort
+
+    new_holdings = {}
+    ClusterOverlap.new(Cluster.where("ht_items.item_id": item_id).first).each do |o|
```

Comment:
I see `#each` is implemented in `ClusterOverlap`. If you `include Enumerable` in that class, you'll get all those methods for free (e.g., map, any?, etc.)

Reply:
:+1: